### PR TITLE
Update Storybook UI theme to improve readability

### LINKED ISSUE #56: [Meta] Rename this repo. Maybe "react-cfpb"?
- [x] Rename repo
- [x] Update SBL Frontend to use new package/repo
- [ ] Fix [Github Pages deployment](https://cfpb.github.io/design-system-react/) of Storybook site

### LINKED ISSUE #136: [Enhancement] Use Avenir Next for Storybook text content
Is it possible to use our brand font in the component library for the text content (documentation)? If yes are there pros and cons related to doing this? Benefits and trade-offs?

### LINKED ISSUE #156: Change all story names to sentence case
This should be applied globally.

Example: Instead of "Default Checkbox" => "Default checkbox"
```
export const DefaultCheckbox: Story = {
  name: 'Default checkbox',
  ...
};
```

### PR DESCRIPTION
**Pull request by meissadia**
Closes #136 
Closes #156 
Closes #56 

## Changes
  - Use `Avenir Next` as default font for all Storybook text
  - Sentence case for story names
  - Allow user to switch between light/dark Storybook theme
  - CFPB logo for branding

## How to test this PR

1. Check out the screenshots below
2. `yarn && yarn start` -> Poke around, see how things feel: are they readable? 

## Screenshots
|Light theme|Dark theme|
|---|---|
|Screenshot 2023-09-05 at 11 40 28 AM|Screenshot 2023-09-05 at 11 40 15 AM|

### CODE REVIEW

**Comment by meissadia** [comment]
I'm not understanding why the netlify deploy of Storybook is failing above?? 
 
`yarn build-storybook` which translates to the same `storybook build -o dist-storybook` that is run in netlify and succeeds locally...

**Comment by contolini** [comment]
@meissadia I scope creeped this PR a bit by adding a commit that moves the repo to Yarn v3. Yarn v1 has issues with ES modules apparently.

File: .storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  textInverseColor: '#ffffff',
+
+  // Sidebar
+  appBg: '#101820', // Background
+  app: '#101820', // Background
+  colorPrimary: '#ff9e1b', // Folder icon
+  colorSecondary: '#0072ce', // onHover, Story icon, Active panel (ex. Addons)
+
+  // Toolbars
+  barTextColor: 'white'
```
Comment (contolini) [inline .storybook/theme.js:21]: How would you feel about installing the [dark mode addon](https://github.com/hipstersmoothie/storybook-dark-mode) to let users control if they want a dark or light background? The [theme configuration](https://github.com/hipstersmoothie/storybook-dark-mode#configuration) looks pretty similar.

Comment (meissadia) [reply .storybook/theme.js:21]: Seems sensible, good catch.

Comment (meissadia) [reply .storybook/theme.js]: Updated to implement light/dark theme switching. Screenshots in [PR description](https://github.com/cfpb/design-system-react/pull/159#issue-1862029102).

**Pull request merged**